areas/data.go

```diff
@@ -0,0 +1,13 @@
+package areas
+
+// AreaDetails represents a response area model from the areas api
+type AreaDetails struct {
+	Code          string                              `json:"code,omitempty"`
+	Name          string                              `json:"name,omitempty"`
+	DateStarted   string                              `json:"date_start,omitempty"`
+	DateEnd       string                              `json:"date_end,omitempty"`
+	WelshName     string                              `json:"name_welsh,omitempty"`
+	GeometricData []map[string]map[string]interface{} `json:"features"`
```

Comment:
This has been updated to ignore the `features` key:

`GeometricData map[string]interface{} `json:"geometry"``

https://github.com/ONSdigital/dp-areas-api/blob/1ba932ffc59b7d56f2eadc9d025a7e6c31f388e8/models/area.go#L68